### PR TITLE
Awesome books: with ES6 🥇 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# What is done in this project 🎟️ 

- [ ] Use ES6 syntax.
- [ ] Divide your code into modules.
- [ ] Create a separate file for each module in the modules directory.
- [ ] Import modules into the index.js file.
- [ ] Create a new directory and initialize it with git.
- [ ] Use npm init -y command to create package.json file.
# Skills used 🧠 

- [ ] Use ES6 modules to write modular JavaScript.
- [ ] Learn how to use proper ES6 syntax.
- [ ] Understand improvements that ES6 brings to JavaScript.
- [ ] Use npm as software packages system.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <li>
+        <a href="#contact" id="contact-link">Contact</a>
+      </li>
+       
+  </nav>
+  </header>
+  
+  <div class="date" id="date"></div>
+
+  <section class="list-section" id="list">
```
Comment (jasonfelice) [inline index.html:34]: - [ ] Kindly make sure this page (section) is visible by default. Currently, I have to click on the `List` to render this section. It'll make more sense if this is visible by default.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+ <center> <section id="add-book">
+    <center><h2>Add a new book</h2></center>
+    <form id="book-form" method="post">
+      <label for="title"></label>
+      <input id='title' type="text" placeholder="title" required> <br>
+      <label for="author"></label> <br>
+      <input id='author' type="text" placeholder="author" required> <br> <br>
+      <button type="button" id="add-btn">Add</button>
+    </form>
```
Comment (jasonfelice) [inline index.html:47]: - [ ] [Optional] To make this application more interactive, consider adding some kind of alert on this page.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @ismailmunyentwari9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Great use of ES6 modules✔️
No Linter errors✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @ismailmunyentwari9 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ No linters error
✔️ Good PR  description

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**